schemas/process.yml

```
@@ -61,7 +61,10 @@
       format: string
       level: extended
       type: long
+      short: Deprecated parent process' pid.
       description: >
+        Deprecated for removal in next major version release. This field is superseded by `process.parent.pid`.
```

```suggestion
        Deprecated for removal in the next major version. This field is superseded by `process.parent.pid`.
```

I think we can drop the word `release` and keep the same meaning.

---

Works for me!